config/settings.yml

```diff
@@ -883,9 +883,9 @@
     :host: 20.minutes
     :vm: 20.minutes
   :concurrent_requests:
-    :historical: 1
-    :hourly: 1
-    :realtime: 20
+    :historical: 250
+    :hourly: 250
+    :realtime: 250
```

Comment:
This is a VMWare specific number, but it's stored here globally.  Is it possible to store it in VMware specific settings?  I wonder if we should move this section to the provider settings entirely.  @agrare Thoughts?

Reply:
So, this is defined in [vmware settings](https://github.com/ManageIQ/manageiq-providers-vmware/blob/master/config/settings.yml#L11) for:

```
:ems:
  :ems_vmware:
    :capture_batch_size: 250
```

Can we just delete these?

Up until now, we have never sent more than 1 record into the queue, so this limiter has not been used.

Now we limit the number of records going into the queue for this limit.

There is some merit to using different limits for the 3 types of collection, but it seemed like a feature that was introduced but never used.